draft-ietf-jsonpath-base.md

```diff
@@ -1188,9 +1195,8 @@ the comparison is between values of the same type which are both numbers or both
       lower Unicode scalar value than the second string or if both strings start with the same Unicode scalar value and
       the remainder of the first string compares less than the remainder of the second string.
 
-Note that `==` comparisons between a structured value (that is, an object or an array) and any value, including the same structured value, yield false
-and `!=` comparisons between a structured value and any value, including the same structured value, yield true.
-<!-- issue: comparison with structured value -->
+Note that comparisons using any of the operators `<`, `<=`, `>`, or `>=` yield false if either value being
```

Comment:
(this may sound contrary to some of my previous arguments, but here goes)

Do we want to allow `<=` and `>=` to consider the `=` part so that if `a==b` yields true, then `a<=b` and `a>=b` yield true as well?

Reply:
Very possibly, but I'd want to be consistent and do the same for booleans and `null`. I think this is separable from the current PR, so I'm going to deem it out of scope for issue 236 (noted [here](https://github.com/ietf-wg-jsonpath/draft-ietf-jsonpath-base/issues/236#issuecomment-1210645442)). I want to make issue 236 and this PR minimal to focus the discussion.